### ISSUE TITLE
'Only save file to disk' still prompts for workspace name

### ISSUE THREAD

**Issue by rikedyp** [Dyalog/ride#495]
### Describe the issue you are having
Using )ed file://plaintext.file pops up window and can edit
On leaving (escape), prompt appears asking how to save the file. Choosing 'Only save file to disk' gives the same prompt as the first option (save in workspace)

### Did you connect to an already running interpreter or start the interpreter from RIDE?
Start from RIDE

### How do you reproduce the issue?
See above

### Paste the contents of Help → About (Shift+F1)
```
IDE:
  Version: 4.1.3366
  Platform: Linux x86_64
  Date: 2018-07-17 15:26:35 +0100
  Git commit: 0088ce492350a8d582b019adfc5d5a87efbdc00b
  Preferences:{
    "kbdLocale":"en_GB",
    "keys":"{\"FX\":[\"Ctrl-S\"]}",
    "pfkeys":"[\"\",\"\",\")vars<ER>\",\")fns<ER>\",\")ops<ER>\",\"\",\"\",\"\",\"\",\")obs<ER>\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\",\"\"]",
    "zoom":"7",
    "floating":"1",
    "floatSingle":"0"
  }

Interpreter:
  Version: 17.0.34137
  Platform: Linux-64
  Edition: Unicode/64
  Date: Aug 31 2018 at 10:58:46

```

**Comment by e9gille** [comment]
This is a bug in the interpreter. I'll report it.

**Comment by johndaintree** [comment]
Interpreter side fix committed to 18.0 and 18.1

**Comment by e9gille** [comment]
Tested on 18.0.40300
